frontend/src/components/RecommendationCard/index.js

```diff
@@ -0,0 +1,41 @@
+import { PropTypes } from "prop-types";
+import "./recommendationCard.scss";
+
+export default function RecommendationCard(props) {
+  const { /*id,*/ headline, status, /*date,*/ score, onClick } = props;
+
+  const handleClickRecommendation = (event) => {
+    onClick && onClick(event, props);
+  };
+
+  const style = () => {
+    return {
+      borderBottomColor:
+        status === "implemented"
+          ? "#0be881"
+          : status === "unapplicable"
+          ? "#ffa801"
```

Comment:
Meinst du wir sollten solche Farben zentral verwalten? Oder ist das Overkill für so ein kleines Projekt?

Reply:
Doch doch, absolut einverstanden. Würde ich erstmal in die Common Config mit aufnehmen 👌